library/ntc_install_os.py

```diff
@@ -155,16 +160,27 @@
 PLATFORM_IOS = 'cisco_ios_ssh'
 PLATFORM_EAPI = 'arista_eos_eapi'
 PLATFORM_JUNOS = 'juniper_junos_netconf'
+PLATFORM_F5 = 'f5_tmos_icontrol'
+PLATFORM_ASA = 'cisco_asa_ssh'
+
+
+def already_set(boot_options, system_image_file, kickstart_image_file,
+                **kwargs):
+    volume = kwargs.get('volume', "")
```

Comment:
why is volume defaulting to an empty string instead of `None`?

Reply:
TY, it's now fixed.